components/FormFieldSelect/FormFieldSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+import { Select } from '../Select';
+
+interface FormFieldSelectItem {
+  value: string;
+  label: string;
+}
+
+interface FormFieldSelectProps {
+  label: string;
+  options: FormFieldSelectItem[];
+  themeColor: 'light' | 'dark';
+}
+
+const FormFieldSelect: FunctionComponent<FormFieldSelectProps> = ({ label, options, themeColor }) => {
+  return (
+    <>
+      <label htmlFor="topic" className="form-label">
+        {label}
+      </label>
+      <div className="form-field__select-dropdown">
```

Comment:
This div is not required. If you review the existing instances of this, you'd see that. It's purpose is when using the default select, and we're using our own Select component.

Reply:
Removed the unnecessary div